docs/resources/mws_workspaces.md

```diff
@@ -50,7 +50,7 @@ resource "databricks_mws_networks" "this" {
   account_id   = var.databricks_account_id
   network_name = "${var.prefix}-network"
   vpc_id       = var.vpc_id
-  subnet_ids = [var.subnet_public, var.subnet_private]
+  subnet_ids = var.subnets_private
   security_group_ids = [var.security_group]
```

Comment:
should this be `var.security_groups` as well and expects a list variable, similar to above, as Databricks can have access to more than a single security group?